cloud-access/access-levels-permissions.mdx

```diff
@@ -33,6 +33,23 @@ If you have multiple AWS accounts and want one of them to define all the permiss
 
 Connections to **AWS Athena**, **AWS CloudFormation**, and **AWS Compute Optimizer** are configured in the same way as AWS connections.
 
+### Connecting to multiple AWS accounts
+
+OpenOps supports two approaches for working with multiple AWS accounts.
```

Comment:
It's up to the user to decide whether to use separate connections or a single connection with assume-role, it's not necessarily related to their AWS organization structure. Even if the accounts are completely separate, the user may still prefer to go with the assume-role approach.

Reply:
@maor-rozenfeld Thanks, but I'm not sure what your comment applies to. Do you suggest that I change some of the wording? I've re-read, and I see nothing that implies that choosing one approach over the other is tied to an organization structure.

Reply:
I think I was referring to the phrasing "If each AWS account is fully independent", you can simply remove it.